### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by kalepail**

### CODE REVIEW

**Review by chadoh — APPROVED** [review]

**Review by ElliotFriend — APPROVED** [review]
So good!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
+
+# RustRover
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+#.idea/
```
Comment (anataliocs) [inline .gitignore:21]: Trailing newline

```suggestion
#.idea/

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # soroban-test-examples
-Soroban Test Examples
+Example contracts for integration testing
```
Comment (anataliocs) [inline README.md:2]: Is there a concept of unit tests vs integration tests in Soroban?

**Review by anataliocs — APPROVED** [review]

**Comment by kalepail** [comment]
Need to mod the repo settings slightly in order to
A) allow PR merges without any CI/CD runs
B) allow PR approvals from the Aha folks

I've got an ops ticket in for those changes and will then get this merged

**Pull request merged**